src/components/settings/LocationSettings.js

```diff
@@ -100,14 +100,16 @@ export const LocationSettings = () => {
         <RegularText>{texts.settingsContents.locationService.alternativePositionHint}</RegularText>
       </Wrapper>
       <Collapsible collapsed={!showMap}>
-        <Map
-          locations={locations}
-          onMapPress={({ nativeEvent }) => {
-            setSelectedPosition({
-              ...nativeEvent.coordinate
-            });
-          }}
-        />
+        <View style={styles.mapContainer}>
+          <Map
+            locations={locations}
+            onMapPress={({ nativeEvent }) => {
+              setSelectedPosition({
+                ...nativeEvent.coordinate
+              });
+            }}
+          />
+        </View>
```

Comment:
opening the area shows me a too small map, is this intended?

<img src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/6d2b7cf3-31b3-47a3-9b58-b38bf2171274" width="200" />
<br/><br/>

and trying to drag around does not work. it behaves strange and does not react correctly on my interactions (not very visible on the video, as my touches are not included):

https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/57b58a20-1efc-4571-af84-56d33a8421c2

Reply:
it should show the whole map, there is some kind of bug caused by the collapsible option, if we'd display the map component by default without the collapsible for example this wouldn't happen

Reply:
ok remove the collapsible and add a bold text in front of the text with the title of the button, that is used now to open the area.
the button itself and the "abbrechen" when opened can be removed with this as well.

Reply:
now the rendering issue caused by the collapsible is fixed but there is still this issue where the pin does move but the map stays static and once we close the map and reopen it it all of a sudden works perfect.
just cant wrap my head around it maybe you'll have a fix for this one.

https://github.com/user-attachments/assets/8d2449ee-226f-476d-b7c8-fc01389c76a1

Reply:
the video does not show the requested change with removing the collapsible and the other things mentioned before. so what to check?

Reply:
there is no collapsible anymore, just similar behaviour handled completely differently.

Reply:
i see. this was not discussed :(
but now keep it, we do not have time to get into it again.

@ardasnturk please check the problem @d4v3d4vE is asking help for.

Reply:
another point is, that when closing the area and opening again, the whole map gets reinitialized and loads again for some time. this should not happen. once loaded it needs to stay loaded.

Reply:
I solved the problem and added the collabsable structure again.

https://github.com/smart-village-solutions/smart-village-app-app/pull/1154/commits/a465ce7e87004b3810f9245288d27ad9d5deb404

Reply:
nice, and when closing/opening the area the map stay loaded where it was before.

Reply:
very nice fix!